components/buttons.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from "react";
```

Comment:
It's better to use the singular because it's only one button: `button.tsx`

Reply:
ok, done